vars/Ubuntu_jammy.yml

```diff
@@ -0,0 +1 @@
+Debian_bookworm.yml
```

Comment:
Can you please include a comment in here explaining why we are doing this?

Reply:
Where exactly do you want the comment?  I can't add it to the symlink...

Reply:
Haha, whoops- wherever you think it will fit.  I'm fine putting it in `molecule/default/tests/test_default.py` where you made the other change in this PR, but I'll defer to your judgement for the best way to remind our future selves about what's going on here.

Reply:
Please see #53 and commit 664fc42.

Reply:
Perfect, thanks!